core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -33,8 +33,10 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   customEnvironmentVariables: Map[String, String],
                                   descriptorPath: Option[Uri],
                                   extraArgs: List[String],
+                                  sourceWorkspaceId: Option[WorkspaceId],
                                   workspaceId: Option[WorkspaceId],
-                                  sourceWorkspaceId: Option[WorkspaceId]
+                                  autodeleteThresholdMin: Option[Int],
```

Comment:
```suggestion
                                  autodeleteThreshold: Option[Int],
```
I believe (based on feedback to your question in the doc) that we are going to drop the `Min` suffix altogether. I will make the suggestions through this PR to make them easier to apply.